setup.py

```diff
@@ -38,4 +54,7 @@
         'console_scripts': ['apertium-apy=apertium_apy.apy:main'],
     },
     packages=find_packages(),
+    cmdclass=dict(
```

Comment:
The dictionary literal form (as above) is much more idiomatic. The `dict` initializer is usually used only on existing vars, e.g. a list of tuples one wishes to convert to a map.